samples/spring-security-basic-auth/src/main/java/sample/spring/xsuaa/SecurityConfiguration.java

```diff
@@ -63,4 +68,8 @@ BearerTokenResolver getTokenBrokerResolver() {
 		return converter;
 	}
 
+	@Bean
```

Comment:
I guess you've already checked that, that this XsuaaServiceConfiguration bean is needed here!?

Do you know which condition is not met so that this [auto-configuration](https://github.com/SAP-samples/cloud-security-xsuaa-integration/blob/master/spring-xsuaa/src/main/java/com/sap/cloud/security/xsuaa/autoconfiguration/XsuaaAutoConfiguration.java#L43) is not considered in this sample?

If yes, I would appreciate a comment here in line 71 that explains why this is needed here :-)